user/forms.py

```diff
@@ -6,36 +9,57 @@
 
 class CustomUserCreationForm(UserCreationForm):
     """
-    This class represents a form the create a new user.
+    A form to create a new user.
     """
     class Meta(UserCreationForm.Meta):
+        """
+        Metadata for the model User. Selects which fields should be editable in this form.
+        """
         model = User
         fields = UserCreationForm.Meta.fields + ('email', 'first_name', 'last_name', 'organisation', 'type_of_use',)
 
 
 class CustomUserChangeForm(UserChangeForm):
     """
-    This class represents a form to change attributes of a user. The Form is used in the admin area.
+    A form to change attributes of a user. This form is only used in the admin area.
     """
     class Meta(UserChangeForm.Meta):
+        """
+        Metadata for the model User. Selects which fields should be editable in this form.
```

Comment:
Wording: I think it should be "Defines which fields..." instead of "Selects...".